packages/wdio-cli/src/launcher.js

```diff
@@ -97,40 +97,51 @@ class Launcher {
      * run without triggering onPrepare/onComplete hooks
      */
     runMode (config, caps) {
-        /**
-         * if it is an object run multiremote test
-         */
-        if (this.isMultiremote) {
-            return new Promise((resolve) => {
-                this.resolve = resolve
-                this.startInstance(this.configParser.getSpecs(), caps, 0)
-            })
-        }
-
         /**
          * fail if no caps were found
          */
-        if (!caps || !caps.length) {
+        if (!caps || (!this.isMultiremote && !caps.length)) {
             return new Promise((resolve) => {
                 log.error('Missing capabilities, exiting with failure')
                 this.interface.updateView()
                 return resolve(1)
             })
         }
 
+        /**
+         * avoid retries in multiremote mode
```

Comment:
It might be a typo. Below you check watch mode instead of multiremote

Reply:
Yeah, typo in comment, thank you

Reply:
@CrispusDH fixed